lib/domain/app_state/room/create_new_group_chat_state.dart

```diff
@@ -13,8 +13,9 @@ class CreateNewGroupChatLoading extends Success {
 
 class CreateNewGroupChatSuccess extends Success {
   final String roomId;
+  final List<String>? invite;
```

Comment:
why we need the `list of invite` here?

Reply:
he need it to create the name for group, but I think it's not a good idea. How about the roomName directly?